src/colorprint.cc

```diff
@@ -89,7 +89,11 @@ std::string FormatString(const char* msg, va_list args) {
 
   std::size_t size = 256;
   char local_buff[256];
+#ifdef __ANDROID__
+  auto ret = ::vsnprintf(local_buff, size, msg, args_cp);
```

Comment:
Yeah, I would rather see the `__GLIBCXX__` check in there as well. As you mentioned this works fine with libc++ :-D